build_library/build_image_util.sh

```diff
@@ -298,6 +298,9 @@ finish_image() {
     sudo fstrim "${root_fs_dir}/usr" || true
   fi
 
+  # Build the selinux policy
+  sudo chroot ${root_fs_dir} bash -c "cd /usr/share/selinux/mcs; semodule -i *.pp"
```

Comment:
it's not the only thing that is going to suck for moving to other archs, dracut also screws us :(